templates/consumer.conf.erb

```diff
@@ -0,0 +1,172 @@
+# The settings in this file are all commented by default, and the commented settings show the
+# default values that Pulp Consumer Client will choose if not specified here.
+
+# The pulp server configuration
+#
+# host:
+#   The pulp server hostname
+# port:
+#   The port providing the RESTful API
+# rsa_pub:
+#   The pulp server public key used for authentication.
+# api_prefix:
+#   The REST API prefix.
+# verify_ssl:
+#   Set this to False to configure the consumer not to verify that the server's SSL cert is signed
+#   by a trusted authority
+#   WARNING: Setting this to False is not safe if you wish to use Pulp with real passwords or
+#            any other private information. It is strongly recommended to keep this set to True, and
+#            to use certificates that are signed by a trusted authority on the web server that
+#            serves Pulp.
+# ca_path:
+#   This is a path to a file of concatenated trusted CA certificates, or to a directory of trusted
+#   CA certificates (with openssl-style hashed symlinks, one certificate per file).
+
+[server]
+host: <%= scope.lookupvar('pulp::consumer::host') %>
```

Comment:
@cristi1979 do you require 2.7 (since you are a prime user of this stand alone) ? We (the Katello team) have been removing 2.7 support from our other modules due to this (the scope[] support) and because our katello-installer doesn't support 2.7 anyway.